scripts/build.py

```diff
@@ -75,6 +90,59 @@ def ipynb(self) -> str:
         return f"{self.basename}.ipynb"
 
 
+class TutorialWebpage:
+    def __init__(
+        self,
+        ipynb_path: str,
+        title: str,
+        description: str,
+        order: int,
+        exclude_output: bool,
+    ) -> None:
+        self.ipynb = ipynb_path
+        self.title = title
+        self.description = description
+        self.order = order
+        self.exclude_output = exclude_output
+
+    def markdown_path(self) -> str:
+        return os.path.join(
+            BUILD_DIR, f"{os.path.splitext(os.path.basename(self.ipynb))[0]}.md"
+        )
+
+    def get_header(self) -> str:
+        return HEADER_TEMPLATE.format(
+            title=self.title, description=self.description, order=self.order
+        )
+
+
+def parse_web_yml(tutorial_dir: Optional[str]) -> List[TutorialWebpage]:
+    with open(WEB_YML, "r") as f:
+        web_config = yaml.safe_load(f)
+    tutorial_webpages = []
+    for i, cfg in enumerate(web_config["tutorials"]):
```

Comment:
is the ordering set alphabetically by default? where can the ordering be enforced (if via website, that works too)

Reply:
just by order in yaml